.ci/.e2e-tests-for-elastic-agent.yaml

```diff
@@ -21,9 +21,10 @@ SUITES:
       # - name: "System Integration"
       #   tags: "system_integration"
       #   platforms: ["debian_arm64", "debian_amd64", "sles15"]
-      # - name: "Stand-alone"
-      #   tags: "stand_alone_agent"
-      #   platforms: ["fleet_elastic_pkg"]
+      - name: "Stand-alone"
```

Comment:
I'm going to revert this commit and add it in a follow up, as the stand-alone tests are failing, even locally

I think we are messing how the elastic-agent Docker container is bootstrapped when running with Docker provider.